composer/callbacks/checkpoint_saver.py

```diff
@@ -303,11 +304,24 @@ def epoch_checkpoint(self, state: State, logger: Logger):
                 logger,
             )
 
-    def get_state_dict(self, state):
-        return {
-            'state': state.state_dict(),
-            'rng': reproducibility.get_rng_state(),
-        }
+    def state_dict(self) -> Dict[str, Any]:
+        state_dict = super().state_dict()
```

Comment:
I think this can be removed -- we dont do it anywhere else and the parent has no state dict functionality

Reply:
I think I copied from EMA - but agreed 😛